misc/short-pitch.md

```diff
@@ -0,0 +1,15 @@
+Software Carpentry's mission is to teach foundational computing skills to
+researchers of all disciplines. Our goal is to help spread skills that both
+provide immediate value to research workflows and help learners build a mental
+framework for thinking about working collaboratively in open and reproducible
+ways. Founded in 1998, we are a non-profit volunteer organization with over 500
+instructors in more than 30 countries, who delivered more than 280 hands-on
```

Comment:
30 countries. In 2015 alone, we delivered more than 280 hands-on workshops to over 10,000 learners.